inst/WORDLIST

```diff
@@ -109,6 +109,7 @@ DualSimulationsSummary
 EffFlexi
 Effloglog
 Effmodel
+efficacies
```

Comment:
is that a real word?... maybe better use `efficacy values`  in text?

Reply:
I checked in the dictionary and `efficacies` is ok but I can change it to `efficacy values`.